storage/rocksdb/rdb_utils.h

```diff
@@ -289,6 +289,9 @@ std::string rdb_hexdump(const char *data, const std::size_t data_len,
                         const std::size_t maxsize = 0)
     MY_ATTRIBUTE((__nonnull__));
 
+// Return dir + '/' + file
+std::string file_in(const std::string &dir, const std::string &file);
```

Comment:
Maybe have a more descriptive name that is consistent with the other helper functions?  Maybe rdb_concat_filepaths?

Reply:
Renamed to `rdb_concat_paths` (let me know if you prefer `rdb_concat_filepaths` or `rdb_concat_file_paths` instead)

(the reason I've been avoiding the `rdb_` prefix was that I thought that the `myrocks::` namespace did the same job)